topostats/run_topostats.py

```diff
@@ -245,6 +245,7 @@ def main(args=None):
         LOGGER.error(error)
 
     # Summary Statistics and Plots
+    summary_config = None
```

Comment:
An alternative is to use the `if config["summary_stats"]["run"]:` that we have an follow it with...

```
else:
    summary_config = None
```

Reply:
Yeah I wasn't sure which to use and went with the option that added fewer lines, but now I think about it, this is more readable. Thanks 😄